client/src/components/common/error-modal.vue

```diff
@@ -2,14 +2,11 @@
   <v-dialog v-model="showModal">
     <v-card>
       <v-card-title class="d-flex justify-space-between">
-        <div>Name Request Encountered an Error</div>
-        <v-btn icon large class="dialog-close" @click="hideModal()">
```

Comment:
Did Tracey ask for this?

Reply:
Sorry i should have mentioned, yes she did. 
As well as nudging the bullet point a little.